src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -22,7 +22,7 @@
         "symfony/dependency-injection": "~3.3",
         "symfony/config": "~3.3",
         "symfony/event-dispatcher": "~3.3",
-        "symfony/http-foundation": "~3.1",
+        "symfony/http-foundation": "~3.3",
```

Comment:
thanks, I was just wondering as the code changes didn't indicate any need for this change